### ISSUE TITLE
HealEntityEvent not working for players base health regeneration?

### ISSUE THREAD

**Issue by Sago92** [SpongePowered/Sponge#2345]
Dear Sponge-Team,

I am currently migrating my Spigot plugin to Sponge and figured out that the HealEntityEvent didn't get fired, when a player regenerates health. Is this event currently not working or isn't it triggering for the base player health regeneration?

**---EDIT 1---**
The event also get not triggered when a player consumes a health potion

SpongeForge Version: 1.12.2-2838-7.1.7-RC3884
SpongeApi Version: 7.1.0

Regards
Sascha

**Comment by gabizou** [comment]
@Sago92 correct, it hasn't been implemented. An oversight on our end! I'll see what I can whip up to implement it in a rough manner.

**Comment by Sago92** [comment]
Any time schedule for the implementation?
